src/vm/eeconfig.cpp

```diff
@@ -789,10 +792,18 @@ HRESULT EEConfig::sync()
         g_fEnableARM = TRUE;
     }
 
-    int forceGCconcurrent = CLRConfig::GetConfigValue(CLRConfig::UNSUPPORTED_gcConcurrent);
-    if ((forceGCconcurrent > 0) || (forceGCconcurrent == -1 && g_IGCconcurrent))
+    bool gcConcurrentWasForced = false;
+#ifdef FEATURE_CORECLR
```

Comment:
We definitely should allow the option to run GCStress with concurrent GC enabled - at the very least, you should be able to turn on/off concurrent GC for GCStress 3. 

what is the order of evaluation supposed to be here? complus overwrites startup flag which overwrites config?